DataLayer/BatteryFaultsData/I_BatteryFaultsData.h

```diff
@@ -38,8 +38,8 @@ class I_BatteryFaultsData : public QObject
     virtual unsigned short getErrorFlag() const = 0;
 
     /* BatteryFaults Sets */
-    void setPackageID(const unsigned char& packageID) = 0;
-    void setErrorFlag(const unsigned char& errorFlag) = 0;
+    virtual void setPackageID(const unsigned char& packageID) = 0;
```

Comment:
A constant in the header such as:
`const unsigned char packageID = X`

or as a namespace?